lib/SPIRV/SPIRVToLLVMDbgTran.cpp

```diff
@@ -256,10 +256,10 @@ SPIRVToLLVMDbgTran::transTypeComposite(const SPIRVExtInst *DebugInst) {
   DICompositeType *CT = nullptr;
   switch (Ops[TagIdx]) {
   case SPIRVDebug::Class:
-    CT = Builder.createClassType(
-        ParentScope, Name, File, LineNo, Size, Align, 0, Flags, DerivedFrom,
-        DINodeArray() /*elements*/, nullptr /*VTableHolder*/,
-        nullptr /*TemplateParams*/, Identifier);
+    CT = Builder.createReplaceableCompositeType(
```

Comment:
I suggest to leave a `TODO` comment here to revert this change once `DIBuilder` is fixed